shared/getNumberOfPeople.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Function which returns the appropriate translation depending on the number from `referralsCount`. 
+ * It is necessary to translate the "people" into e.g. Polish, where the word "people" declines differently 
+ * depending on whether number of people is:
+ * 1) (case nr 1) between 5-21 (inclusive 5 and 21) and if number ends with digit:
+ *  - 0 or 1 
+ *  - between 5-9 (inclusive)
+ *  - between 11-19 (inclusive)
+ * 2) (case nr 2) number 2, 3 or 4 and if number end with digit 2, 3 or 4
+ */ 
+
+export const getNumberOfPeople = (
+    numberOfPeople, 
```

Comment:
numberOfPeople: number

Reply:
Resolved in 5482e6b0ad979ce6f0f89bb295128bf7365d09fa